modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -242,6 +243,8 @@ public ObjectEntry addObjectEntry(
 			_objectDefinitionPersistence.findByPrimaryKey(objectDefinitionId);
 
 		_validateGroupId(groupId, objectDefinition.getScope());
+		_validateWorkflowAction(
+			null, objectDefinition, serviceContext.getWorkflowAction());
```

Comment:
Can we add this validate to after `_validateValues`? To follow the same pattern in update.